core/lib/multivm/src/versions/vm_latest/bootloader_state/l2_block.rs

```diff
@@ -56,12 +56,6 @@ impl BootloaderL2Block {
         self.txs_rolling_hash = concat_and_hash(self.txs_rolling_hash, tx_hash)
     }
 
-    pub(crate) fn interim_version(&self) -> BootloaderL2Block {
-        let mut interim = self.clone();
```

Comment:
Nit: It doesn't look like `Clone` impl for `BootloaderL2Block` is used any more (other than in `Clone` derive in `BootloaderState`, which is also unused). Ditto for the fast VM. IMO, could make sense to remove these derives to not create possibility for performance degradation in the future.